core/src/epicli/data/common/ansible/playbooks/roles/opendistro_for_elasticsearch/tasks/install-es-RedHat.yml

```diff
@@ -2,8 +2,7 @@
 - name: Install Open Distro for Elasticsearch packages
   yum:
     name:
-      - java-1.8.0-openjdk-headless
```

Comment:
Is JDK installed somewhere else?

Reply:
We are using elasticsearch-oss, no elasitcsearch-oss-no-jdk. JDK is bundled and used by default. I did some briefly checks and it looks like we dont need to install additional java. During tests it may turn out otherwise.

Reply:
We are using elasticsearch-oss, no elasitcsearch-oss-no-jdk. JDK is bundled and used by default. I did some briefly checks and it looks like we dont need to install additional java. (During tests it may turn out otherwise)